src/components/AssetTypes/AssetTypesComponent.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Form } from 'semantic-ui-react';
+import PropTypes from 'prop-types';
+import ArtButton from '../common/ButtonComponent';
+import InputFluid from '../common/TextInputComponent';
+import DropdownComponent from '../common/DropdownComponent';
+
+const placeMakesInSemanticUIOptions = props =>
+  props.map((option, index) => ({
+    key: index,
+    text: option.sub_category_name,
+    value: option.id
+  }));
+
+const AssetTypesComponent = props => (
+  <div>
```

Comment:
`<div>` is unnecessary as `<form>` encapsulates all the markup

Reply:
I will change this.